tests/test_doc_archives.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import pytest
+
+from aqt.archives import QtArchives
+
+
+@pytest.mark.parametrize("os_name,version,target,datafile", [
+    ('windows', '5.15.2', 'doc', 'windows-5152-src-doc-example-update.xml')
+])
+def test_parse_update_xml(monkeypatch, os_name, version, target, datafile):
```

Comment:
Please change test function name other than the name of test in test_archive.py.

Reply:
Uhm, it is not hazardous from TravisCI result.